operations/payments.md

```diff
@@ -95,7 +95,7 @@ Adds a new external payment to a bill of the specified customer. An external pay
 | `ClientToken` | string | required | Token identifying the client application. |
 | `AccessToken` | string | required | Access token of the client application. |
 | `Client` | string | required | Name and version of the client application. |
-| `CustomerId` | string | required | Unique identifier of the [Customer](customers.md#customer). |
+| `AccountId` | string | required | Unique identifier of the [Customer](customers.md#customer) or [Company](companies.md#company). |
```

Comment:
Let's start with the strikethrough deprecation where we remove the deprecated field from the example, however we **keep** it in the table marked as deprecated and referencing to the deprecation page. The main reason why we were doing it this way (removing altogether) was that we didn't have the deprecation+discontinuation process in place and we didn't want partners to adopt deprecated fields/endpoint. It's a common pattern visible not only in our other API documentation (booking engine) but outside the company as well. We agreed to follow this approach with @MikeAdamsMews and @vitasamek so let's start doing it. There is no point in adding already removed deprecated fields but let's say that from now we'll keep the deprecated fields. @MikeAdamsMews please confirm the format of the deprecation we go with. cc @JordSips